lib/plugins/login/login.js

```diff
@@ -0,0 +1,132 @@
+'use strict';
+
+const BbPromise = require('bluebird');
+const crypto = require('crypto');
+const readline = require('readline');
+const opn = require('opn');
+const fetch = require('node-fetch');
+const jwtDecode = require('jwt-decode');
+const chalk = require('chalk');
+const getFrameworkId = require('../../utils/getFrameworkId');
+const updateConfig = require('../../utils/config/update');
+
+const config = {
+  AUTH0_CLIENT_ID: 'iiEYK0KB30gj94mjB8HP9lhhTgae0Rg3',
+  AUTH0_URL: 'https://serverlessinc.auth0.com',
+  AUTH0_CALLBACK_URL: 'https://serverless.com/welcome',
+};
+
+function base64url(url) {
+  return url.toString('base64')
+    .replace(/\+/g, '-')
+    .replace(/\//g, '_')
+    .replace(/=/g, '');
+}
+
+class Login {
+  constructor(serverless, options) {
+    this.serverless = serverless;
+    this.options = options;
+
+    this.commands = {
+      login: {
+        usage: 'Login to your Serverless platform account',
+        lifecycleEvents: [
+          'login',
+        ],
+      },
+    };
+
+    this.hooks = {
+      'login:login': () => BbPromise.bind(this).then(this.login),
+    };
+  }
+  openBrowser(url) {
+    let browser = process.env.BROWSER;
+    if (browser === 'none') {
+      return false;
+    }
+    if (process.platform === 'darwin' && browser === 'open') {
+      browser = undefined;
+    }
+    try {
+      const options = { app: browser };
+      opn(url, options).catch(() => {});
+      return true;
+    } catch (err) {
+      console.log('---------------------------'); // eslint-disable-line
```

Comment:
you could use `this.serverless.cli.log` or `this.serverless.cli.consoleLog` to get rid of the eslint disables.